Yoast/yoast-components
# Upgraded webpack to 4.7

## Summary

I upgraded the development app to use webpack 4, since this needed to happen some time in the future anyway and I got an error when I used `yarn install` because of an incompatible package with node 9.

This PR can be summarized in the following changelog entry:

* Upgrades webpack to version 4+

## Relevant technical choices:

* Removed JSON-loader
* Set webpack mode to `development`

## Test instructions

This PR can be tested by following these steps:

* Install dependencies
* Make sure the development app works as expected

Fixes #
